Product/Production/Services/PatientCorrelationCore/src/test/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/proxy/PatientCorrelationProxyWebServiceUnsecuredImplTest.java

```diff
@@ -77,6 +81,18 @@ public void testretrievePatientCorrelations() throws Exception {
             any(AssertionType.class));
     }
 
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testAddPatientCorrelationPLQ() throws Exception {
+
+        PatientCorrelationProxyWebServiceUnsecuredImpl impl = getPatientCorrelationProxyWebServiceUnsecuredImpl();
+        impl.addPatientCorrelationPLQ(mockPLQrec, mockAssertion);
+        when(mockWebProxy.getUrlLocalHomeCommunity(NhincConstants.PATIENT_CORRELATION_SERVICE_NAME)).thenReturn("");
```

Comment:
You need to specify your "When"s before you actually invoke your service in order for the mocks to work, which begs the question as to whether or not this is doing what it should, and if it is needed.